### ISSUE TITLE
Bouton "Importer"

### ISSUE THREAD

**Issue by gbert1** [dvoitsek/recolnat-lab#128]
Je propose de rajouter un bouton "Importer" en 2ième position dans le SM entre "Nouveau set" et "Ouvrir".
Puis de modifier la boîte pour avoir 2 boîtes : 

1/ Une boîte "Nouveau set et ajouts depuis Explore"

2/ Une boîte aves les imports CSV.

**Comment by dvoitsek** [comment]
Nous avons déjà passé trop de temps sur la boîte d'import. Il y a des fonctionnalités plus importantes à implémenter.

**Comment by gbert1** [comment]
Je n'ai pas du tout été impliqué sur cette boîte :-) 
Mon but est de rendre visible l'import au niveau 1 du SM.
Ce que je propose prend un quart d'heure, peut-être 20mn :-) c'est dispatcher ton code.

**Comment by dvoitsek** [comment]
Tu as fait l'ergonomie de cette boîte et je l'ai implémentée selon ton ergonomie.

**Comment by gbert1** [comment]
Effectivement, mais jeudi j'ai reconstaté ce que j'avais déjà constaté ces derniers temps, c'est que l'import est invisible au niveau 1. Mon ergo a besoin d'une évolution :) Il faut rendre visible explicitement l'import au niveau 1. 
Je te propose une solution moins coûteuse de ton temps : changer l'intitulé du bouton comme suit : 

Ajout ou Nouveau set
via Explore ou Import CSV

Ce  n'est pas ma solution préférée mais elle permet l'essentiel.
